src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteBuildConfiguration.java

```diff
@@ -838,7 +838,7 @@ public RemoteBuildInfo updateBuildInfo(@Nonnull RemoteBuildInfo buildInfo, @Nonn
 		}
 
 		// Only avoid url cache while loop inquiry
-		String buildUrlString = String.format("%sapi/json/?seed=%d", buildInfo.getBuildURL(),
+		String buildUrlString = String.format("%sapi/json/?tree=result&seed=%d", buildInfo.getBuildURL(),
```

Comment:
Hi @sbeaulie ,
In fact, looking to your proposal, here, you forgot the building field: https://github.com/jenkinsci/parameterized-remote-trigger-plugin/blob/020aa67fe2d3b0bf553dca7226ca0dd647caad31/src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteBuildConfiguration.java#L849

So, it should be tree=result**,building**&seed=%d

This is certainly why your PR was failing during validation.

But I like your idea & this will be a very good optimization as the api/json can sometimes be very big depending on git changes list or archived files & others Build details. This should optimize greatly network & memory usage.

Best regards
Louis

Reply:
Hi @cashlalala 

Tried with my fix above about the building missing field,
Compiled with my jdk1.8.0_201 & apache-maven-3.6.2, unittests passed well.
tried the PRT plugin on my own Jenkins 2.205 & was running smooth

@sbeaulie 

I think that you can add this missing field & that should do the trick ;)

Best regards
Louis

Reply:
Just also did this change locally as proposed by Louis and all works well. We have in our Jenkins a plugin that stores the artifacts not on the master but in a blob store. So every time the api is called Jenkins needs to ask our blob store about which artifacts are stored there. This increases additionally the time to calculate the final result and sometimes leads to running into the hard coded timeout of 10 seconds for GET requests (the timeout is also something that could be made configurable)
Would be